### DISCUSSION TITLE
Is there a way to port an existing WinUI 3 application to an Gtk application without shared project?

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#9094] [Q&A]
[_Question from discord_](https://discord.com/channels/372137812037730304/550416151172087808/989259950180155392)

**Comment by jeromelaban** [comment]
Yes, it is possible as you can take all the files in the shared project and put them in the Skia.Gtk project. Yet, if you do this, you'll lose (or make it more difficult) the ability to use those files in other platforms.

**Reply by ArchLeaders** [reply]
Nice. Thanks!
